R/tex-utils.R

```diff
@@ -107,6 +107,8 @@ tex.catwitherror <- function(x, dx, digits=1, with.dollar=TRUE, human.readable=T
     }
 
     if(!is.na(err) && err > 0) {
+      N <- absolute.number.digits(err, digits)
```

Comment:
could we add a comment here that also reference the `borderline_error` test? Otherwise the logic seems a little ad hoc and smells like an accidental drop of precision.

Reply:
Sure.